dali/operators/reader/loader/video_loader.cc

```diff
@@ -690,8 +690,9 @@ void VideoLoader::read_file() {
       break;
     } else if (dec_status == VidReqStatus::REQ_ERROR) {
       LOG_LINE << "Request failed" << std::endl;
-      DALI_FAIL("Detected variable frame rate video. The decoder returned frame that is past "
-                "the expected one");
+      DALI_FAIL("The decoder returned frame that is past the expected one."
```

Comment:
I wonder if it make sense to use `make_string`.

Reply:
Done